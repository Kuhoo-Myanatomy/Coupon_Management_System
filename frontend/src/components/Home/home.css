html{
    margin: 0px;
    /* background-image: url("https://images.unsplash.com/photo-1561715276-a2d087060f1d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1170&q=80"); */
    background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url(https://images.unsplash.com/photo-1561715276-a2d087060f1d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1170&q=80);
    height: 100%;
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover;
    background-attachment: fixed;
}
h1{
    color: white;
}
.main{
    display: flex;
    margin-top: 10%;
    flex-direction: column;
    justify-content: center;
    align-items: center;
}
.footer{
    background-color: black;
    position:fixed;
    bottom: 0px;
    width: 100%;
    display: flex;
    justify-content: center;
    padding: 5px;
    text-align: center;
    align-items: center;
}
p {
    margin-top: 0;
    margin-bottom: 1rem;
    color: gainsboro;
}
a{
    text-decoration: none;
}
.main button{
    display: block;
    width: 80%;
    background-color: #4CAF50;
    color: white;
    font-size: 18px;
    padding: 10px;
    border-radius: 5px;
    border: none;
    cursor: pointer;
}
  
.main button:hover {
    /* background-color: #3e8e41; */
    color: white;
}