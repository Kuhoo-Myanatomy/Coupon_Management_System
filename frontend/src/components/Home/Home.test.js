import Home from './Home';
import{shallow,configure} from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';

configure({ adapter: new Adapter() });

describe("Testing Home Component",()=>{
    it("Matching Home's snapshot",()=>{
        const component=shallow(<Home/>);
        expect(component).toMatchSnapshot();
    });
});