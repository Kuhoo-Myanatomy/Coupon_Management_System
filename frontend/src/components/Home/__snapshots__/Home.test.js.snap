// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Home Component Matching Home's snapshot 1`] = `ShallowWrapper {}`;
